python/lsst/ctrl/bps/drivers.py

```diff
@@ -249,8 +249,12 @@ def submit_driver(config_file, **kwargs):
     config_file : `str`
         Name of the configuration file.
     """
-    wms_workflow_config, wms_workflow = prepare_driver(config_file, **kwargs)
-    submit(wms_workflow_config, wms_workflow)
+    _LOG.info("Starting submission process")
+    with time_this(log=_LOG, level=logging.INFO, prefix=None, msg="Completed entire submission process"):
+        wms_workflow_config, wms_workflow = prepare_driver(config_file, **kwargs)
+        submit(wms_workflow_config, wms_workflow)
```

Comment:
Need the starting line and time_this line for the submit stage (otherwise the timings for the stages do not add up to the complete submission process time).

Reply:
Got it. Will fix.